client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -89,10 +89,18 @@ struct SubscribeConfig {
   std::string id;
   // If set to false then all certificates related to subscription will be ignored
   bool use_tls;
+  // TRC/S certificate path
+  std::string trsc_tls_cert_path;
+  // URL used to decrypt private keys
+  std::string secrets_url;
   // Buffer with the client's PEM encoded certififacte chain
   std::string pem_cert_chain;
   // Buffer with the client's PEM encoded private key
   std::string pem_private_key;
+  // TRC ID obtained from OU field in pem_cert_chain
+  std::string id_from_cert;
```

Comment:
Not sure if this provides any value either. If the user can fill out `id` and `id_from_cert` then we cannot really compare because we don't know if `id_from_cert` actually came from the certificate.

Reply:
Yes, thanks. This shouldn't be configurable. Updated.